src/react/hooks/useMediaUIDispatch.js

```diff
@@ -0,0 +1,10 @@
+import { useContext } from 'react';
+import { ReactMediaChromeContext } from '../components/Context.js';
+
+export function createMediaUIDispatchHook(context = ReactMediaChromeContext) {
+  return function useDispatch() {
+    const store = useContext(context);
+    return store.dispatch;
+  };
+}
+export const useMediaUIDispatch = createMediaUIDispatchHook();
```

Comment:
I'm not familiar enough with dispatch/selector functions to have a strong opinion, but `MediaUI` feels odd here for naming. In "use **[blah]** Dispatch" what is blah meant to be describing?

Reply:
this was primarily based on prior convos. Other, generic state mgmt libs such as Redux have a similarly generically named `useDispatch()`. We can certainly also export using the generic name, though it's plausible that folks would have to do some renaming in their module if they (not unlikely) are using both e.g. redux and our media store in the same component/module/etc. Also, I could see folks encountering `useDispatch()` in a codebase and just assuming it's e.g. the Redux version of `useDispatch()`. If we want to keep the "blah" around, this would be more us coming up with a convention that "feels right" given the domain scoping, e.g. `useMediaDispatch()` (which is what it was originally) or `useMediaUIDispatch()` (which is what it became after convos around making it more explicitly about UI media state).